magma/db/migrations/011_add_gnomon_tables.rb

```diff
@@ -0,0 +1,27 @@
+Sequel.migration do
+  change do
+    create_table(:rules) do
+      primary_key :id
+
+      String :project_name, null: false
+      Integer :version_number, null: false
+      column :config, :json, null: false
+      DateTime :created_at, null: false
```

Comment:
I think for the other apps we used the timestamp plugin for Sequel, which also requires a column for `:updated_at`. While we may not use that column, the plugin is quite useful -- should we add the `:updated_at` column anyways?